pkgs/top-level/python-packages.nix

```diff
@@ -1490,6 +1490,9 @@ in {
     inherit (pkgs) sentencepiece pkgconfig;
   };
 
+  tokenizers = disabledIf (!isPy3k)
+    (callPackage ../development/python-modules/tokenizers { });
```

Comment:
`toPythonModule` needs to be used here for withPackages to function.

Reply:
Added.